scipy/optimize/_minimize.py

```diff
@@ -65,6 +65,11 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
         where ``x`` is a 1-D array with shape (n,) and ``args``
         is a tuple of the fixed parameters needed to completely
         specify the function.
+
+        Suppose the callable has signature ``f0(x, *args, **kwargs)``, where ``args``
+        and ``kwargs`` are required positional and keyword arguments. Rather than
+        passing ``f0`` as the callable, consider wrapping it to accept only ``x``;
+        e.g., pass ``fun=lambda x: f0(x, *args, **kwargs)`` as the callable.
```

Comment:
Let me try again, and see if I can clarify the confusion between `args` (in optimization function signature) and `args` in `fun` definition. Maybe I'll try more than once, and you can see which appeal to you, or not.

```suggestion
        Consider, e.g., a callable with signature ``f0(x, k, alpha=0.1)``.
        Since it has an argument and keyword argument, for the purpose of using it
        here, you'd need to wrap it to only accept a single argument, `x`.
        Pass it to this function as, e.g., ``fun=lambda x: f0(x, 3, alpha=0.1)``,
        to fix the argument and keyword argument.
```